### PR TITLE
Add AS-SETs + AS48717, AS203913, AS199762, AS150249

### PR DESCRIPTION
**Pull request by Plactrix**

### CODE REVIEW

**Review by christheradioguy — COMMENTED** [review]
Why is the indentation being changed?

**Comment by Plactrix** [comment]
did not realize it changed the entire file, was meant to just correct 48717, ill correct it

**Comment by Plactrix** [comment]
corrected.

File: clients.yml [hunk 1]
```diff
@@ -1126,6 +1126,39 @@ clients:
   - 192.34.27.115
   - 2602:ffb1:0200:0:192:34:27:115
 
+- asn: "48717"
```
Comment (christheradioguy) [inline clients.yml]: What is this ASN? It hasn't been active since 2015 - https://bgp.he.net/AS48717

Comment (Plactrix) [reply clients.yml]: AS48717 is my friend's ASN, he said it was in the routing table last week and was announcing something. I can remove it until he resolves it if you'd like

Edit: He said HE hasn't updated that his ASN was routed last week and the data on there is incorrect currently

Comment (christheradioguy) [reply clients.yml]: yeah lets remove since it doesnt look active in any way

Comment (Plactrix) [reply clients.yml]: Updated, included PR #5 as well

File: clients.yml [hunk 2]
```diff
@@ -1126,6 +1126,48 @@ clients:
...
+
+- asn: "150249"
+  cfg:
+    filtering:
+      irrdb:
+        as_sets:
+        - AS150249:AS-ATHARVA
+  description: "PT Atharva Telematika Persada"
+  ip:
+  - 192.34.27.116
```
Comment (christheradioguy) [inline clients.yml]: lets keep the IPv4 and IPv6 addresses in sync - so this should be 192.34.27.119/24

Comment (christheradioguy) [reply clients.yml]: Actually, please assign v4 addresses to each peer, even if they are not using v4, it breaks the website formatting otherwise.

Comment (Plactrix) [reply clients.yml]: Neon corrected it, is fixed now.

**Review by christheradioguy — APPROVED** [review]

**Pull request merged**